src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -64,6 +71,18 @@
 ]
 
 
+def _get_unpad_data(attention_mask):
```

Comment:
This seems like a copied from this method: https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L63 - can you add a `# Copied from` statement similarly as : https://github.com/huggingface/transformers/blob/main/src/transformers/models/falcon/modeling_falcon.py#L111

Reply:
done